src/scripts/DynamicBackground.ts

```diff
@@ -0,0 +1,134 @@
+class DynamicBackground {
+    constructor() {}
+
+    /*
+    SUN & MOON
+    */
+    static setSunMoonPosition = (date = new Date()) => {
+        const w = window.innerWidth / 1.2;
+
+        // Do the same thing with the height. Responsive = Good times.
+        const h = window.innerHeight / 1;
+
+        // Get the hours and minutes.
+        const hours = date.getHours();
+        const mins = date.getMinutes();
+
+        // Calculate the position of the sun and moon based on the time.
+        const sunRad = (((hours) * 60 + mins) / (24.00 * 62.00)) * Math.PI * 2;
+        const moonRad = (((hours + 12) * 60 + mins) / (24.00 * 60.00)) * Math.PI * 2;
+
+        // Calculate the axis
+        const sunX = (w / 1.8) - (w * Math.sin(sunRad)) / 2;
+        const sunY = (h / 2) + (h * Math.cos(sunRad)) / 2;
+        const moonX = (w / 1.8) - (w * Math.sin(moonRad)) / 2;
+        const moonY = (h / 1.4) + (h * Math.cos(moonRad)) / 2;
+
+
+        // Apply the poisitions based on our previous calculations
```

Comment:
`poisitions` spelling error

Reply:
Hmm, someone must be messing with my code 😓